projects/exui-common-lib/src/lib/components/search-location/search-location.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-form-group">
+    <div class="location-picker-custom">
+        <label class="govuk-body" for="location-picker">
+            Search for a location by name
+        </label>
+        <br>
+        <div style="width: 50%; display: inline-block; margin-right: 4px;">
+            <div>
+                <div class="autocomplete__wrapper">
+                    <div style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; margin-bottom: -1px; margin-right: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 1px;"><div id="location-picker__status--A" role="status" aria-atomic="true" aria-live="polite"></div><div id="location-picker__status--B" role="status" aria-atomic="true" aria-live="polite"></div></div>
```

Comment:
This line is too long and hard to read.
1.Can this style move to scss
2 What are these two divs id="location-picker__status--A" and id="location-picker__status--B" for?